draft-zip.md

```diff
@@ -0,0 +1,109 @@
+```
+ZIP: 
+Title: Deposit 60% of transaction fees to the Zcash Sustainability Fund
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits:
+Status: Draft
+Category: Ecosystem
+Created: 2023-09-21
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words “MUST”, “SHOULD”, “SHOULD NOT”, “MAY”, “RECOMMENDED”, “OPTIONAL”, and “REQUIRED” in this document are to be interpreted as described in RFC 2119. [1]
+
+The term “network upgrade” in this document is to be interpreted as described in ZIP 200. [2]
+
+The term “Block Rewards” in this document is to be interpreted as described in ZIP TBD [3]
+
+The term “Issuance” in this document is to be interpreted as described in ZIP TBD [3]
+
+“We” - the ZIP authors, owners listed in the above front matter
+
+“`ZSF_BALANCE_AFTER[h]`” is balance of the Zcash Sustainability Fund as defined in ZIP ###
+
+# Abstract
+
+This ZIP proposes a modification to transaction fees, diverting 60% of transaction fees back into the `ZSF_BALANCE_AFTER[h]`, while  the destination of the remaining 40% is unchanged and goes to the block miner,. This proposal effectively “unmints” a portion of transaction fees, contributing to a deflationary effect and offering long-term support for the Zcash network.
+
+This ZIP attempts to establish a symbiotic relationship between miner incentives and sustained network growth. It achieves this by splitting transaction fees: 40% goes directly to miners, incentivizing them to include transactions, while the remaining 60% is deposited into the  `ZSF_BALANCE_AFTER[h]`. This approach mitigates a "bootstrapping problem", a problem that arises when 100% of transaction fees go to the ZSF and miners are not incentivized to include transactions in block. This ZIP navigates this problem by ensuring miners continue to receive direct rewards for including transactions, while still contributing to the ZSF.
+
+Implementing this change allows the ZSF to accrue value earlier. By ensuring a consistent source of funding, the ZSF contributes to bolstering the Zcash network’s long-term security and sustainability.
```

Comment:
The feedback about using "the ZSF" in most sections, and `ZSF_BALANCE_AFTER[h]` or `ZSF_BALANCE_AFTER[h-1]` in specification sections, also applies here.

Reply:
I've removed the uses of `ZSF_BALANCE_AFTER`.